consensus/ibft/src/main/java/tech/pegasys/pantheon/consensus/ibft/ibftmessagedata/CommitPayload.java

```diff
@@ -66,4 +68,37 @@ public Hash getDigest() {
   public Signature getCommitSeal() {
     return commitSeal;
   }
+
+  @Override
+  public ConsensusRoundIdentifier getRoundIdentifier() {
+    return roundIdentifier;
+  }
+
+  @Override
+  public boolean equals(final Object o) {
+    if (this == o) {
+      return true;
+    }
+    if (o == null || getClass() != o.getClass()) {
+      return false;
+    }
+    if (!super.equals(o)) {
```

Comment:
can the super check be removed?

Reply:
That's autogenerated code. Yeah doesn't technically need it, but I'd rather just leave the intellij code as it